### ISSUE TITLE
Reclassify: magnitude

### ISSUE THREAD

**Issue by nicolevasilevsky** [pato-ontology/pato#326]
Propose moving PATO_0002016 magnitude to child of 'process quality'
and need to define PATO_0002016 magnitude

suggested def: the size or extent of a process

**Comment by dosumis** [comment]
That's seems an oddly narrow definition of magnitude:
![image](https://user-images.githubusercontent.com/112839/87702651-8f80f500-c791-11ea-9291-206377014bf6.png)

**Comment by nicolevasilevsky** [comment]
Do you have a suggested revision?

**Comment by sbello** [comment]
@dosumis Is your concern that this is limited to "process"? I checked the MP and we use increased/decreased magnitude in a few eq (eg enhanced/impaired wound healing)
We typically use size and not magnitude for anatomical entities.
Would it be helpful if the term label were changed to something like "magnitude of process" to match the definition restriction. The current term is undefined but I can see the issue since magnitude can also be used in relation to force or brightness of a sun.
May be it would be better to create a new set of process restricted magnitude terms and leave the existing terms alone?

**Comment by cmungall** [comment]
Yes magnitude is also used as a kind of meta quality to define inc/dec

I’m not sure magnitude should apply directly to characteristics? Always 2nd
order? How is the proposed definition for process magnitude different from
extent or duration?
